src/commands/registry.rs

```diff
@@ -49,6 +49,10 @@ pub struct Push {
     )]
     pub insecure: bool,
 
+    /// Skip composing the application's components before pushing it.
+    #[clap(long = "skip-compose", env = PUSH_ALWAYS_SKIP_COMPOSE)]
```

Comment:
I'm not sure if an environment variable fits well with how this would be used.  I don't think we have a clear philosophy on that though... happy to keep this if you think it's appropriate.

Reply:
Yeah i'm not particularly bound to this; someone mentioned that they would like to disable it by tossing an environment variable in their shell.

Reply:
Updated and removed.